### ISSUE TITLE
Plugin Installing Issues

### ISSUE THREAD

**Issue by JoeMacz** [itsabot/abot#53]
I've been getting 500 on the fetching those plugins.

```
Fetching 3 plugins...
WARN: 500 - 500 Internal Server Error
WARN: 500 - 500 Internal Server Error
WARN: 500 - 500 Internal Server Error
Installing plugins...
Success!
```

This is the config, JSON file:
```

{
	"Name": "Abot",
	"Description": "Digital assistant framework",
	"Version": "0.2.0-alpha",
	"ImportPath": "github.com/itsabot/abot",
	"Dependencies": {
		"github.com/itsabot/plugin_onboard": "*",
		"github.com/itsabot/plugin_weather": "*",
		"github.com/itsabot/plugin_hello": "*"
	}
}

```

Is this is an issue on my end or?

**Comment by egtann** [comment]
Nope, no issue on your end. For now you can ignore those warnings. I'm working on a fix at the moment.

**Comment by JoeMacz** [comment]
Thanks mate!

**Comment by egtann** [comment]
Should be fixed with #64!
